core/src/main/java/org/mule/runtime/core/internal/policy/DefaultPolicyManager.java

```diff
@@ -70,6 +77,9 @@
   public SourcePolicy createSourcePolicyInstance(Component source, CoreEvent sourceEvent,
                                                  Processor flowExecutionProcessor,
                                                  MessageSourceResponseParametersProcessor messageSourceResponseParametersProcessor) {
+
+    logEvent(sourceEvent.getContext(), sourceEvent.getMessage());
```

Comment:
This seems that is going to log the same that the first policy when it starts. Also, this won't be just for policies. We may have to remove it